packages/tar-mirage/tar-mirage.0.9.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "1.2"
+maintainer:   "dave@recoil.org"
+authors:      [ "Dave Scott" "Thomas Gazagnaire" "David Allsopp" ]
+tags:         ["org:xapi-project" "org:mirage"]
+homepage:     "https://github.com/mirage/ocaml-tar"
+bug-reports:  "https://github.com/mirage/ocaml-tar/issues"
+dev-repo:     "https://github.com/mirage/ocaml-tar.git"
+doc:          "https://mirage.github.io/ocaml-tar/"
+
+build: [
+  ["jbuilder" "subst" "-p" name "--name" name] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+
+build-test: [
+  [ "jbuilder" "runtest" ]
+]
+
+depends: [
+  "jbuilder"          {build & >="1.0+beta7"}
+  "ocamlfind"         {build}
+  "tar"
+  "cstruct"           {>= "1.9.0"}
+  "re"
+  "result"
+  "mirage-block-unix" {test & >= "2.5.0"}
+  "mirage-types-lwt"  {>= "3.0.0"}
+  "lwt"
+  "io-page"
+  "mirage-block-unix" {test}
```

Comment:
you have two occasions of `mirage-block-unix` dependencies in here - once `test`, once `test & >= "2.5.0"` -- is this intended?